app/reducers/tabs.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
again, lets do this only for specific lines, and only when it makes sense. Thus far we could just be using an arrow func and returning `null` eg.